test/decl/operator/declared_power_operator.swift

```diff
@@ -0,0 +1,12 @@
+// RUN: %target-typecheck-verify-swift
+
+infix operator **
+
+func **(a: Double, b: Double) -> Double { return 0 }
+
+func testPowerOperatorWithDeclaredPowerOperator() { 
```

Comment:
Thanks--can we simplify this name? It seems redundant to me. `testPowerOperator` or `testDeclaredPowerOperator` seems fine.

Reply:
Fixed! :)